test/test_autograd.py

```diff
@@ -6783,6 +6783,17 @@ def test_leaky_relu_inplace_with_zero_slope(self, device):
         expected = torch.tensor([0., 0., 1.], device=device)
         self.assertEqual(a.grad, expected)
 
+    def test_elu_inplace_with_neg_alpha(self, device):
```

Comment:
Could you mark this test to be `onlyOnCPUAndCUDA`? Pytorch/XLA runtime error won't be catch by this check.

Reply:
Thanks!